### PR TITLE
bpo-46530: add `"thread_time"` to `test_time.test_get_clock_info`

### PR DESCRIPTION
**Pull request by sobolevn**
Originally discovered here: https://github.com/python/typeshed/pull/7040

https://bugs.python.org/issue46530

### CODE REVIEW

File: Lib/test/test_time.py [hunk 1]
```diff
@@ -556,20 +556,26 @@ def test_localtime_failure(self):
...
-            #self.assertIsInstance(info, dict)
-            self.assertIsInstance(info.implementation, str)
-            self.assertNotEqual(info.implementation, '')
-            self.assertIsInstance(info.monotonic, bool)
-            self.assertIsInstance(info.resolution, float)
-            # 0.0 < resolution <= 1.0
-            self.assertGreater(info.resolution, 0.0)
-            self.assertLessEqual(info.resolution, 1.0)
-            self.assertIsInstance(info.adjustable, bool)
+            with self.subTest(name=name)
```
Comment (sobolevn) [inline Lib/test/test_time.py]:
```suggestion
            with self.subTest(name=name):
```

**Review by JelleZijlstra — APPROVED** [review]

**Comment by JelleZijlstra** [comment]
@gvanrossum I think this one can be merged.

**Review by gvanrossum — APPROVED** [review]
Agreed.

**Comment by gvanrossum** [comment]
Backports?

**Comment by sobolevn** [comment]
Failures look unrelated 🤔 

Both backports make sense:
- https://github.com/python/cpython/blob/3.10/Lib/test/test_time.py#L547
- https://github.com/python/cpython/blob/3.9/Lib/test/test_time.py#L547

**Comment by JelleZijlstra** [comment]
Look at you, making the buildbots run out of disk space.

(This one also looks unrelated. The buildbot ran out of space before it even finished compiling the interpreter.)

**Pull request merged**